app/src/main/java/org/fossasia/pslab/experimentsetup/NFETOutputCharacteristicsExperiment.java

```diff
@@ -198,9 +198,9 @@ protected Void doInBackground(Float... params) {
             float voltage = params[0];
             scienceLab.setPV1(voltage);
             float readVoltage = (float) scienceLab.getVoltage("CH1", 10);
-            voltageAxis.add(readVoltage);
-            float resistance = 560;
-            currentAxis.add((voltage - readVoltage) / resistance);
+            voltageAxis.add(Math.abs(readVoltage - 3));
+            float resistance = 56;
```

Comment:
Oops ! That's a mistake. I was playing with the device with some weird values after I've made the PR and forgot that they are still there when I shifted the gain set method call! I'll revert those back and make a quick commit!!!